.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -24,8 +24,8 @@ Put an `x` in the boxes that apply. You can also fill these out after creating t
 them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
 -->
 
-- [ ] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md)
-- [ ] I have read the [security policy](../security/policy)
```

Comment:
Where does `../security/policy` point to?

Reply:
https://github.com/ory/fosite/network/alerts

Just click the link in your PR ;)

Reply:
Oh, I was clicking on it while browsing the repository. OK.